ArdusatSDK.cpp

```diff
@@ -1504,7 +1491,7 @@ const char * Temperature::toJSON(const char * sensorName) {
  * @endcode
  */
 TemperatureTMP::TemperatureTMP(void) :
```

Comment:
I'm _fairly_ sure that this constructor isn't necessary, IIRC, the superclass's `void` constructor gets called automatically in the derived class' default constructor.